Gemfile.lock

```diff
@@ -140,6 +156,13 @@ GEM
     rake (12.0.0)
     ruby_parser (3.8.4)
       sexp_processor (~> 4.1)
+    sass (3.4.23)
+    sass-rails (5.0.6)
+      railties (>= 4.0.0, < 6)
+      sass (~> 3.1)
+      sprockets (>= 2.8, < 4.0)
+      sprockets-rails (>= 2.0, < 4.0)
+      tilt (>= 1.1, < 3)
```

Comment:
`bundle check` didn't get rid of this, should I just clear it manually? Or copy/paste from another branch?